### PR TITLE
Added default keymap for keeb.werk nano.slider

### PR DESCRIPTION
**Pull request by Duckle29**
The reason for the difference in the third layer, is that the third layer in the C implementation is mainly there as a demo layer, showing users how they can make custom keycodes and macros. Since this won't work in the configurator, I added some other functionality. 

The slider currently won't work, but I'm working on moving the slider function to a weak default function for the keyboard.

### CODE REVIEW

File: public/keymaps/k/keebwerk_nano_slider_default.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    [
+      "TO(2)",
+      "RGB_TOG",  "RGB_MOD",  "RGB_VAI",
+      "KC_TRNS",  "RGB_RMOD", "RGB_VAD",  "KC_TRNS"
+    ],
+    [
+      "TO(0)",
+      "KC_VOLD",  "KC_VOLU",  "KC_F24",
+      "KC_MRWD",  "KC_MFFD",  "KC_F23",  "KC_MPLY"
+    ]
```
Comment (noroadsleft) [inline public/keymaps/k/keebwerk_nano_slider_default.json]: On one hand, this doesn't match the keymap that's on qmk_firmware.

On the other, I think this one is better. :/

https://github.com/qmk/qmk_firmware/blob/cf40c33c907c44753bb145b9f2d5107447422fbc/keyboards/keebwerk/nano_slider/keymaps/default/keymap.c#L35-L48

Comment (Duckle29) [reply public/keymaps/k/keebwerk_nano_slider_default.json]: I agree. I'll change the one in the keyboard on my PR on that

Comment (noroadsleft) [reply public/keymaps/k/keebwerk_nano_slider_default.json:21]: `!ping` Are you still planning on/intending to update this on the qmk_firmware side?

**Comment by noroadsleft** [comment]
`!ping` Are you still planning on/intending to update this on the qmk_firmware side?

**Comment by Duckle29** [comment]
Thanks for the ping, this had slipped my mind. Made the changes in a PR in qmk_firmware :) https://github.com/qmk/qmk_firmware/pull/8785

File: public/keymaps/k/keebwerk_nano_slider_default.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "keyboard": "keebwerk/nano_slider",
+  "keymap": "default_4f1a62c",
+  "commit": "4f1a62ca1d8e1cd1528305bda6a315489738c29f",
```
Comment (noroadsleft) [inline public/keymaps/k/keebwerk_nano_slider_default.json:4]:
```suggestion
  "keymap": "default_7873b49",
  "commit": "7873b49d40cef838268575d5b31a1d377b218787",
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**